File: JAVA/Kode Program/Komponen.java
public class Komponen {
    private String merk;
    private int harga;

    public Komponen(String merk, int harga) {
        this.merk = merk;
        this.harga = harga;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public void setHarga(int harga) {
        this.harga = harga;
    }

    public String getMerk() {
        return this.merk;
    }

    public int getHarga() {
        return this.harga;
    }
}

File: PYTHON/Kode Program/Komponen.py
class Komponen:
    def __init__(self, merk="", harga=0):
        self._merk = merk
        self._harga = harga

    def set_merk(self, merk):
        self._merk = merk

    def set_harga(self, harga):
        self._harga = harga

    def get_merk(self):
        return self._merk

    def get_harga(self):
        return self._harga
